### PR TITLE
Notes

### PR DESCRIPTION
**Pull request by MarkVed17**
- Add notes
- Refactored Notes Editor
- Refactored Rich Text Editor
- Pinned & Unpinned notes

### CODE REVIEW

**Review by horsemaker — APPROVED** [review]
The whole code looked really clean! 👌
- Proper Structure
- Understandable & Modular Code

**Review by vijaytembugade — APPROVED** [review]
Good work.
- Proper structure of a code
- Text editor looks good.

File: src/frontend/components/NoteCard/NoteCard.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    const response = await updateNoteService(auth.token, {
+      ...note,
+      color: newColor,
+    });
+    if (response !== undefined) {
+      setNotes(response);
+    }
+  };
+
+  const labelPicker = async (newLabel) => {
```
Comment (t007rushi) [inline src/frontend/components/NoteCard/NoteCard.js:34]: you extract them out as services

File: src/frontend/constants/noteConstants.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const TITLE = "TITLE";
```
Comment (t007rushi) [inline src/frontend/constants/noteConstants.js:1]: like that you make constats nice

**Review by t007rushi — APPROVED** [review]
overall code is well structured

**Pull request merged**